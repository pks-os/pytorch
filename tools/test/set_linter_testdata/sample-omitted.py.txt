ignored = set()  # noqa: set_linter
a = set()
b = 'set()'
c = set
ignored = set()  # noqa: set_linter
d = c.set
f = (
   set(
   )
)
ignored = (
   set(  # noqa: set_linter
   )
)
e = """ set()
set() set x.set set()
\""""

# From here, every mention of set is ignored

class A:
    def set(self, x):
        self.x = x

set = A().set  # Also gets ignored!

good = {}
bad = {1}  # TODO: Should get caught
