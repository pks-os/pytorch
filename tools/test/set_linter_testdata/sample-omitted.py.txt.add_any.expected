from torch.utils._ordered_set import OrderedSet
ignored = set()  # noqa: set_linter
a = OrderedSet[Any]()
b = 'set()'
c = OrderedSet[Any]
ignored = set()  # noqa: set_linter
d = c.set
f = (
   OrderedSet[Any](
   )
)
ignored = (
   set(  # noqa: set_linter
   )
)
e = """ set()
set() set x.set set()
\""""

# From here, every mention of set is ignored

class A:
    def set(self, x):
        self.x = x

set = A().set  # Also gets ignored!

good = {}
bad = {1}  # TODO: Should get caught
